### PR TITLE
optional username/ssh step for iso on update, update wiz sends ssh/user

### PR DESCRIPTION
**Pull request by resoluteCoder**
This should complete this task/bug [RHELPLAN-91235](https://issues.redhat.com/browse/RHELPLAN-91235)

### CODE REVIEW

File: src/components/form/ReviewStep.js [hunk 1]
```diff
@@ -108,14 +108,14 @@ const ReviewStep = () => {
...
-          ''
-        ) : (
+        {getState().values.imageType.includes('rhel-edge-installer') ? (
           <ReviewSection
             title={'Registration'}
             data={registration}
             testid={'review-image-registration'}
           />
+        ) : (
+          ''
```
Comment (boaz0) [inline src/components/form/ReviewStep.js:118]: null

Comment (resoluteCoder) [reply src/components/form/ReviewStep.js:118]: Agreed

**Review by matoval — APPROVED** [review]
Works great! 👍🏽

**Comment by boaz0** [comment]
merged :bomb:

**Pull request merged**